editor/scene/canvas_item_editor_plugin.cpp

```diff
@@ -1802,7 +1802,7 @@ bool CanvasItemEditor::_gui_input_resize(const Ref<InputEvent> &p_event) {
 					};
 
 					DragType resize_drag = DRAG_NONE;
-					real_t radius = (select_handle->get_size().width / 2) * 1.5;
+					const real_t radius = select_handle->get_size().width * (1.5f / 2.0f);
```

Comment:
Adding `const` changes are unrelated to changes to literals, and should be in a different PR imo (I wouldn't approve them personally, which is why I haven't approved the PR).